lib/utils/tests/getFileExtension-test.js

```diff
@@ -0,0 +1,31 @@
+import { expect } from 'chai';
+import {
+  describe,
+  it,
+} from '@bigtest/mocha';
+
+import { getFileExtension } from '..';
+
+describe('getFileExtension', () => {
+  it('should return file extension', () => {
+    const testFile = new File([], 'test.csv');
+
+    expect(getFileExtension((testFile))).to.be.equal('.csv');
```

Comment:
```suggestion
    expect(getFileExtension((testFile))).to.equal('.csv');
```
Please, check other places